Services/CO.CDP.Organisation.WebApi.Tests/UseCase/UpdateConnectedEntityUseCaseTest.cs

```diff
@@ -0,0 +1,169 @@
+using System;
+using System.Threading.Tasks;
+using AutoMapper;
+using CO.CDP.Organisation.WebApi.Model;
+using CO.CDP.Organisation.WebApi.UseCase;
+using Persistence = CO.CDP.OrganisationInformation.Persistence;
+using Moq;
+using Xunit;
+
+namespace CO.CDP.Organisation.WebApi.Tests.UseCase;
+
+public class UpdateConnectedEntityUseCaseTests
+{
+    private readonly Mock<Persistence.IConnectedEntityRepository> _mockConnectedEntityRepository;
+    private readonly Mock<Persistence.IOrganisationRepository> _mockOrganisationRepository;
+    private readonly Mock<IMapper> _mockMapper;
+    private readonly UpdateConnectedEntityUseCase _useCase;
+
+    public UpdateConnectedEntityUseCaseTests()
+    {
+        _mockConnectedEntityRepository = new Mock<Persistence.IConnectedEntityRepository>();
+        _mockOrganisationRepository = new Mock<Persistence.IOrganisationRepository>();
+        _mockMapper = new Mock<IMapper>();
+
+        _useCase = new UpdateConnectedEntityUseCase(
+            _mockConnectedEntityRepository.Object,
+            _mockOrganisationRepository.Object,
+            _mockMapper.Object);
+    }
+
+    [Fact]
+    public async Task Execute_ThrowsUnknownOrganisationException_WhenOrganisationDoesNotExist()
+    {
+        var command = (organisationId: Guid.NewGuid(), connectedEntityId: Guid.NewGuid(), updateConnectedEntity: new UpdateConnectedEntity
+        {
+            EntityType = (ConnectedEntityType)0
+        });
+        _mockOrganisationRepository.Setup(repo => repo.Find(command.organisationId)).ReturnsAsync((Persistence.Organisation)null);
```

Comment:
Static analysis reports a problem here: `Converting null literal or possible null value to non-nullable type.`.

Reply:
Thanks @jakzal . Have pushed an update to address this.